src/CommonITILObject.php

```diff
@@ -236,6 +236,17 @@ public function __set(string $property_name, $value)
                 // TODO 10.1: throw exception instead
                 trigger_error("Readonly field: '$property_name'", E_USER_WARNING);
                 break;
+
+            default:
+                if (version_compare(PHP_VERSION, '8.2.0', '<')) {
```

Comment:
As said on internal discussion channel, depreciation should only occurs on 10.1, not on 10.0.x with any PHP version

Reply:
Deprecation was already trigerred by PHP 8.2 in GLPI 10.0.9. We should not hide it.